core/embed/rust/src/ui/layout/obj.rs

```diff
@@ -208,6 +208,14 @@ impl LayoutObj {
         unsafe { Gc::as_mut(&mut inner.root) }.obj_paint()
     }
 
+    /// Place but do not paint.
```

Comment:
What are the benefits of supporting this operation?

Reply:
I am doing this so the `layout::page_count` is being set and I can read it correctly - as without `place()`, `Rust` layouts cannot determine their `page_count`.

Not sure why it is not necessary in `TT`

Reply:
I could not find any calls to `place` caused by `attach_timer_fn` - where is it done?

I tried not calling `layout.place` before `RustLayout::page_count` and it resulted in a lot of UI errors - the issues are surprisingly only with the headers, where it does not count with the scrollbar (it should create a marquee) - so some issue with `page_count` in `Rust` - in `ScrollableFrame::place`

![image](https://github.com/trezor/trezor-firmware/assets/42543243/a5a781d6-292a-480a-a477-383d61071a3c)

Reply:
`ui_layout_attach_timer_fn` -> `LayoutObj::obj_event` -> `Child<T>::obj_place` -> `Component::place`

Maybe it's a bug in the header placement code, that goes away when you call place() twice?

Reply:
@mmilata you were right, I had a problem in that special `place` that was "fixed" by being called twice. Thanks a lot for the idea!

Got rid of this in 62317d2ee2d9dfd6905503f126e4a4ea2a3efbfd